### DISCUSSION TITLE
Process <defunct>

### DISCUSSION THREAD

**Discussion by paulocotta** [nginx-proxy/nginx-proxy#1702] [General]
Hi guys, how are you?
I would like to check if the process question is normal - defunct:

> root 3884455 0.0 0.0 0 0 pts/0 Z+ 11:33 0:00 [generate-dhpara] <defunct>
https://github.com/nginx-proxy/nginx-proxy/issues/1161
This is normal, I'm asking, as this didn't happen in previous versions
Tks a lot

**Comment by ansidev** [comment]
@paulocotta https://serverfault.com/questions/390202/ubuntu-displays-there-is-1-zombie-process-upon-login
Hope this help!
